dockerfile

```
@@ -0,0 +1,40 @@
+# Download dependencies
+RUN go mod download
+
+# Copy the rest of the application source code
+COPY . .
+
+# Build the application for Linux
+# Based on your taskfiles, the output is in the bin directory
+RUN CGO_ENABLED=0 GOOS=linux go build -v -ldflags="-s -w" -o bin/ePrometna_Server .
+
+# Stage 2: Create the final runtime image
+FROM alpine:latest
```

Hadolint (2.12.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)